core/src/main/scala/kafka/server/KafkaServer.scala

```diff
@@ -21,11 +21,10 @@ import java.io.{File, IOException}
 import java.net.{InetAddress, SocketTimeoutException}
 import java.util.concurrent._
 import java.util.concurrent.atomic.{AtomicBoolean, AtomicInteger}
-
```

Comment:
sorry nit here again 😅

Reply:
I will fix this on Monday. It seems that my IDE is doing this automatically somehow.